notebooks/243-tflite-selfie-segmentation/243-tflite-selfie-segmentation.ipynb

```diff
@@ -0,0 +1,716 @@
rko+i0Qw20vRnS88VYX3u8pzW0mUXZNaPth8OQsYXLZ9zqDyJlfrSZr0ISEFHakFiaRmpeW6DsOyDN3ufHhgYGBgYGBgYGBgY6OMJYmyUlz5xR2FeorGz+63ORtyWNMeqbWuEEjy2OW4/6M7URcnmQOTrfZ6X4hguuV1xxPjWqwv+L3/7V/Ff/pPfx4evzvjud4C/+rM/gf/9f/jH8dkbjz/5/e/hJ7/0Hr78tQ/gOAIgfP2DO/zf/j+/gb/1K18B44L/+c/+KP6tH/8CPBH+zn//m/jP/t6v4Fe+doc5ML703OF/8m/9KP53/+GfxA985hb/0z//U/jbv/SvAb488GWSM0lj32y3hA3bM+3ueN0suHxxb71co+8y5gC6BD3QvJlTE5CTkByrLPNr8injdt2VE6c+N/+Y75Z1a0u1xOkalLzuWZHYIK/L89LO5cyoreTf9lHiEuMilGmvvdUve5MfLf8+uiMVSOSnvUZD2BGKO6Z1HCOFV89OZfb6bFHlnEBXoXq8dYbGMas1WQCzaNGO0OLFLs9StJYK10DFOvWesrtO77o7SjQiF8/w5TLsudz8XrsCmZXVHM+7+Js53nXvJBemlgqRGef5gst8ATNjOomg4X1ygxTLPSO7YQLX7aR1OAglJinXw5DQLQ5m+xCJTMd3CAyiCAcn8Zkywa0BsSM4ujTP5VxnhI0QZrx+fYfXr19hvpwlrgEzyInrsclPmdzXvlcXR56ByIQYNb5AEp1UmFARhTtjhQjzfJH7VSLJdSOF/hwhux8KFYQiM0KQGCMhBjBLsHRrOZHdOC0Z492hU7mbyvO8SRMZM88giJCSx1MzJ8ocodIXLNZ+HGOyO9O+lZSRGRSjbD4whHrvfiXDhqs8inCWqXaxGlGBY/vynwy6lFL3maRNW9znKYhpcc1FJKHFPbAP83z0gGce1YV40bd1fq2rsd06beS1WJq2LpPaVkMSJwcGBgYGBgYGBgYGBt4uHi1sZFcfLc32oLfYJTWVCkm/uRy3fmwy4Svfsnmxzjk3L6oO7kEvnFJLSj9S3i//zrfwN37uX+PX34+ITPjGizv8Nz/3L/E/+7M/iu/64qfx2VuP7/n0M3ztq1+HS5TCOTC+/OEZv/2Nl/jxLz7HH/2Bz+Fzn7rB/X3Ay1czvvT5T+NTn/ssKAKnecbr+ws+uJ/xPe/d4qd+4DP40S99Bt/84BX4qndJeRllIPmYjyDT5n0qpXxXcqlJcOWkamdUK8RM9VKO3ElHCIhdMIOjMy/vDHEd1Cfp7XFWqiduxaio6I/F58qtFzpJO5fQEz8Kac7VcSVK87G1z4bo74pUHWsbQtMHXYZ+2QuLXcHp2DVzi2Kpb45BYFyeHMmJYPv9OOQyc2PlUUGuN7EK86M7qduRuzTsUSJ3Y/zm8ksebfrS8s0c40IoXXP5tUDTGYVVXitkvPm+DPFCRrtriK4uebo85jJBqm4P03hp2j3L3xwxzzPuz/e4XC4gEE63pyxq2DgTxBIXoJC2KOI3lYDaxRJl0QyLdivGV8ZyZ8kJHsDBE1TQV3I21UkuSdxGqpAox2q3RjFKwAVKNxahwOW78/0dXr56gfvzfRawGCJsEBFubiZ4n0QHV8a0ihiIYmHBnAhvjcll2o3Nv2oZqH0eY6pfTlPEsSymXdusmygdpUHMbVB0KzTrfxZrVqxNoiwKtD9lKsl8tGviHAKcdwAhB+aWdiaxYoFWnXNwcBUXhPjn7IqtrbPeMcvTQkt4L89Jg8kId5JGXbw9kWHZoWlAQHL/tfPUwOaXWdfl/5UzD4gl19R1DQtBdSvtwfrsVkc1Oc1xLdvUYAv3oG9LuRoYGBgYGBgYGBgYGDB4tLDhUoDI1f3DR192mLeTtm9aRfXIgob9ss1Ld6jmFAsS6jrEoqrg9//gBb764jWcfw4KhIAbfOPDe7z/8h4EwIMweYdAQPCFXJidA3vgU+9M+Mx7t/De4XQL/I/+zR/Bn/2TPwj1CEIMOAd88dPPAACffu7wQ5/7FN5//1WPD9wEA+AY5cfsuJTvrrGvqbHk6tZzW/L8hN0d4WzP7EM2/vZJ2JbAzsRYcg1S+StnrBAKHecW+SV/WV7lAWtlvC0v24gahbfTHBekWj6rsbwhbsiwDfDe3MuVbQlv+1Uh+bpc+eKEMktpzcfPFSCOxwiehaiDakBySuIWg4CrX/KxELB1hm2R6+3WVkTH6R4har/Ouu/VzUgg6qzbVxPvXAQoHefMcEpc72XYWOEVF3DLMc5mlz5VF27mTUobY8TlfCmiBhFONyJqEFEJnJ3WQzAwEYkoYPvb7F5v65ibjNTffV3vOoaR/DhaUq7tunXUNdU6asaRWUUWcWskLutiZbUhpDYDade+rovgKNYad6/w6tVHmOeL6aOySDkvBHaIEY48yAEcGwHASfvqTn5mpUhlnOjyoU3ovcMlELw/waf4HWUZ1zroHHtYm621db4/GtFHhZVsmZJamhw1Gs1xsllF0srllLYLSVt59pjDnNsqxoDLReo4TRMAnzeZ9AXF2tJFr0GErHactmOzfLJfZTGRy90+ay9p+XTeS1s9hcWGaZMj97WqzC2xwp4CscioQrktsu3P705m1UdeHu7mfzUOD/v9VNysX6vpOrk93M3owMDAwMDAwMDAwMDAw/F4YWPy8N3dzVfm8+gczMtm59uywzr9zTG9wD3stdK+pp/ngDkGgGZMmFLwVYcYa7GFnUdIZRIDnhkTEzATEGQX5RwjvvX6Hh/cXRDJIXLy0R0jfv+D12ByeA2Hj16ccYPiKqS6DLNzskeq2l2hD3oX7e2u7jfSJxuk119TUXsbzan3R0/ZsB8JXea59d0t+0w7eSjjZ/82x2vrjocQfPtqhMQZWZLNLWF2hHQStyiJmIwHiJQVQSfn19kx3U8VqyOc/6nTxQ5zoxYCSWIyWscWCwZQQwja78p5KdekiG2RS0oIl2zCQ1SNWux7AhCKSCI+9q+xiivEe92
```

Comment:
Maybe have another dropdown here? The **NOTE** can be a warning if the user picks Webcam as a source.

Reply:
We can also check available cameras for the user and put them in the dropdown.

Reply:
I'm not sure that it is needed, first of all there is some cameras which not supported to opencv  and user can do not want to use web cam at all. I'm trying to keep it aligned with other notebooks and the same approach used in yolov8 notebook